tests/framework-tests/core/html/Keyboard-a11y-test.html

```diff
@@ -14,9 +14,17 @@
 
     <!--  These are the jqUnit test js files -->
     <script type="text/javascript" src="../../../lib/qunit/js/qunit.js"></script>
+    <script src="/testem.js"></script>
```

Comment:
Would be neater to move the inclusion of `testem.js` down below to be with other files that are added for the coverage client.